releasenotes/notes/apm-single-span-extractor-41d255ab50ca60e6.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    APM : Adds single span extractor to process single span ingestion from tracers
```

This makes sense to me and you, but the change log should make sense for the user. Maybe we should write something like:
```suggestion
    APM : Added support for tracer single span sampling. See DataDog/dd-trace-go#1357.
```
"extractor" is a very implementation specific detail.